scalyr

```diff
@@ -624,7 +631,7 @@ if __name__ == '__main__':
     parser.add_argument('command', choices=all_commands.keys(),
                         help='specifies the action to be performed')
     parser.add_argument('--version', action='version', version='%(prog)s ' + TOOL_VERSION)
-    parser.add_argument('--server', default='https://www.scalyr.com',
+    parser.add_argument('--server', default='',
```

Comment:
Could you remove the default='', so this line just looks like:

    parser.add_argument('--server',

If a default isn't specified, then the default value be None, and I think it's better to use None if no value is specified rather than an empty string.

The reason for this is that if testing against an empty string and someone runs the command from bash:

    ./scalyr --server "$SCALYR_EU_SERVER" .....

But they forget to actually set $SCALYR_EU_SERVER in their shell, then it will evaluate to `./scalyr --server "" ...` which is an empty string, and so it would then silently connect to the `https://www.scalyr.com` address rather than the presumed https://eu.scalyr.com

By contrast, if None is the default, then an empty string for the server will fail out with a connection error.

The logic on lines 93-95 will then also need to be changed to check for None rather than ''.